src/Tizen.NUI/src/public/XamlBinding/SizeTypeConverter.cs

```diff
@@ -28,6 +28,12 @@ public override object ConvertFromInvariantString(string value)
                     int z = (int)GraphicsTypeManager.Instance.ConvertScriptToPixel(parts[2].Trim());
                     return new Size(x, y, z);
                 }
+                else if (parts.Length == 2)
```

Comment:
same as above

Reply:
Hello, Mr. SSong, You're right. Thank you for your remind.